### PR TITLE
moonflowers-final review

### PR DESCRIPTION
**Pull request by Mypeterhub**

### CODE REVIEW

**Comment by Mypeterhub** [comment]
Could not get images hiding and click events to show it on the side. One of the image didn't show up and found if in menu file. Changed it from .jbp to jpg and it works.

File: data/menu.json [hunk 1]
```diff
@@ -5,13 +5,13 @@
         "name" : "Assorted Muffins",
         "type" : "bakery",
         "price" : "$1.75",
-        "img" : "muffins.jpg"
+        "img" : "images/muffins.jpg"
```
Comment (mbMosman) [inline data/menu.json]: You could also put this in the JS where you are setting the image source.  Many times you are not able to modify your data.

File: js/script.js [hunk 2]
```diff
@@ -1,36 +1,144 @@
...
+    //coffeeShopApp.createRegisterForm()
+    //function checkRegistation() {
+
+    $("#register").submit(function () {
+        var fullname = $("#fullname").val();
+        var text = $("#email").val();
+        console.log(text);
+
+        if (text === "") {
+            $("#register").before("<p>Please give your email for contact information!</p>");
```
Comment (mbMosman) [inline js/script.js]: You should have looked up how to verify that an actual email was entered.  Something like: http://www.w3resource.com/javascript/form/email-validation.php

**Comment by mbMosman** [comment]
Nice work overall, but email validation is incomplete.

**Pull request closed without merging**